source/server/listener_manager_impl.h

```diff
@@ -115,6 +115,7 @@ using ListenerImplPtr = std::unique_ptr<ListenerImpl>;
   COUNTER(listener_modified)                                                                       \
   COUNTER(listener_removed)                                                                        \
   COUNTER(listener_stopped)                                                                        \
+  GAUGE(total_filter_chains_draining, NeverImport)                                                 \
```

Comment:
Please make a note to document this when you do the docs, release notes, etc. in the final change. Thank you!

Reply:
ack.
Thank you!